compiler/test/dotty/tools/vulpix/TestConfiguration.scala

```diff
@@ -76,4 +77,19 @@ object TestConfiguration {
 
   /** Enables explicit nulls */
   val explicitNullsOptions = defaultOptions and "-Yexplicit-nulls"
+
+  /** Default target of the generated class files */
+  private def defaultTarget: String = {
+    import dotty.tools.dotc.config.Properties.javaVersion
+    val version: Int =
```

Comment:
scala.util.Properties contains `isJavaAtLeast`

Reply:
Thanks for the tip 👍